md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/DQSetupAPI.java

```diff
@@ -165,8 +149,7 @@ RestWrapper list(@RequestParam(value = "page", defaultValue = "0") int startPage
      * @return restWrapper Instance of DQSetup.
      */
     @RequestMapping(value = {"", "/"}, method = RequestMethod.PUT)
-    public
-    @ResponseBody
+    @ResponseBody public
     RestWrapper insert(@ModelAttribute("dqsetup")
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "insert" is 15 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)